<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeDialog</class>
 <widget class="QDialog" name="HomeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>个人中心</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="avatarLabel">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>240</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>240</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 60px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="usernameLabel">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>邮箱地址</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>